test/extensions/filters/udp/dns_filter/dns_filter_test_utils.cc

```diff
@@ -0,0 +1,70 @@
+#include "dns_filter_test_utils.h"
+
+#include "common/runtime/runtime_impl.h"
+
+#include "test/test_common/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace UdpFilters {
+namespace DnsFilter {
+namespace Utils {
+
+std::string buildQueryForDomain(const std::string& name, uint16_t rec_type, uint16_t rec_class) {
+
```

Comment:
delete blank line. Please check the entire PR.

Reply:
I grepped the filter and tests several times for variations on this pattern and didn't find anything else.  Apologies for missing this last one.